config.yml

```diff
@@ -17,6 +17,7 @@ defaults:
       clientId: !env TASKCLUSTER_CLIENT_ID
       accessToken: !env TASKCLUSTER_ACCESS_TOKEN
     queueBaseUrl: !env QUEUE_BASE_URL
+    botName: 'taskcluster[bot]'
```

Comment:
This should be in `app`. In `defaults` it should be `!env BOT_USERNAME`, in prod it should be `taskcluster[bot]` and in staging it should be `taskcluster-staging[bot]`.